lib/pdf_fill/forms/common_ptsd.rb

```diff
@@ -60,6 +61,8 @@ def expand_unit_assigned_dates(incident)
       end
 
       def split_approximate_date(date)
+        # from/to are optional but need to be accounted for
```

Comment:
is this a duplicate comment of line 65, ` # year/month/day are optional and can be XXed out`